src/annotator/test/guest-test.js

```diff
@@ -44,8 +44,8 @@ describe('Guest', () => {
   let DocumentMeta;
   let fakeDocumentMeta;
 
-  let PdfIntegration;
-  let fakePdfIntegration;
+  let PDFIntegration;
+  let fakePDFIntegration;
```

Comment:
The capitalization here was corrected to match the real class. See related Slack discussion: https://hypothes-is.slack.com/archives/C1M8NH76X/p1616742301079700.